corehq/apps/case_search/xpath_functions/ancestor_functions.py

```diff
@@ -104,10 +92,7 @@ def _child_case_lookup(context, case_ids, identifier):
     """
     es_query = context.helper.get_base_queryset().get_child_cases(case_ids, identifier)
     context.profiler.add_query('_child_case_lookup', es_query)
-    if _should_scroll(context):
-        return es_query.scroll_ids()
-    else:
-        return es_query.get_ids()
+    return es_query.get_ids()
```

Comment:
Is this correct? Going by the name it should be `scroll_ids()` if this PR releases the scroll ids